root/components/list/AreaList.js

```diff
@@ -29,25 +31,30 @@ const AreaList = ({
   $c,
   areas,
   checkboxes,
+  mergeForm,
   order,
   sortable,
 }: Props) => {
   const columns = React.useMemo(
     () => {
-      const checkboxColumn = $c.user_exists && checkboxes
-        ? defineCheckboxColumn(checkboxes)
+      const checkboxColumn = $c.user_exists && (checkboxes || mergeForm)
+        ? defineCheckboxColumn(checkboxes, mergeForm)
         : null;
       const nameColumn =
         defineNameColumn<AreaT>(l('Area'), order, sortable);
       const typeColumn = defineTypeColumn('area_type', order, sortable);
+      const removeFromMergeColumn = mergeForm
```

Comment:
Would it make sense to check if `areas.length > 2` here, so an empty column isn't shown in that case? Or check that in `defineRemoveFromMergeColumn` instead so it applies to all?

Reply:
I thought defineRemoveFromMergeColumn did that already, but maybe I messed up. I'll check!

Reply:
Yes, defineRemoveFromMergeColumn already does this.

Reply:
Sorry, thought I checked but must've missed that somehow!